.gitignore

```diff
@@ -42,7 +42,7 @@ __pycache__/
 /src/libcore/unicode/SpecialCasing.txt
 /src/libcore/unicode/UnicodeData.txt
 /src/libcore/unicode/downloaded
-/target/
+target/
```

Comment:
Is this intentional?

Reply:
Yes; I should have put it in a separate commit. RLS puts the `target/` directory in `src/libcore` for me.